### PR TITLE
API documents added to developer portal

### PR DESCRIPTION
**Pull request by BenjaminAnding**

### CODE REVIEW

File: _data/sidebar.yml [hunk 1]
```diff
@@ -109,6 +109,38 @@ entries:
           version: all
           print: true
 
+    - title: APIs
```
Comment (ahamino) [inline _data/sidebar.yml]: Cloud APIs

File: pages/eaasdocs/AffdexAPIOverview.md [hunk 2]
```diff
@@ -0,0 +1,152 @@
...
+request = Net::HTTP::Get.new uri
+request.basic_auth 'username', 'password'
+request['Accept'] = 'application/json'
+response = http.request request
+
+puts response.body
+end
+```
+
+### Java
```
Comment (ahamino) [inline pages/eaasdocs/AffdexAPIOverview.md]: Let's use bootstrap to create a tabbed view for all the languages,
http://getbootstrap.com/components/#nav

Also, use a code editor to properly format the code. indentation .. etc

File: pages/eaasdocs/AffdexAPIOverview.md [hunk 3]
```diff
@@ -0,0 +1,152 @@
...
+BufferedReader br = new BufferedReader(new InputStreamReader(in));
+String readLine;
+while (((readLine = br.readLine()) != null)) {
+System.out.println(readLine);
+}
+}
+}
+```
+
+### <raw>C#</raw>
```
Comment (ahamino) [inline pages/eaasdocs/AffdexAPIOverview.md]: What is <raw> and why do we need it.

Side note, don't mix .. markdown and html .. If you want to add HTML .. try to use include and include external html file when possible. This to make the source code of the developer portal readable in a code editor. When u put HTML in markdown file .. the HTML is not recognized the by editors language parsers as HTML .. and ends up being unreliable.

File: pages/eaasdocs/AffdexAPIOverview.md [hunk 4]
```diff
@@ -0,0 +1,152 @@
...
+## Index Service Examples  
+
+Sometimes the hardest part of systems integration is establishing a basic connection between the two systems.  These examples connect to Affdex and request the index, so they test http communication, basic authentication, and content negotiation.  Once you can get a simple program like these to work in your language of choice, you've got a solid platform to build from.  
+
+### curl  
+
+curl is a command-line tool for Unix systems, and it's perhaps the simplest way to access a RESTful server.  From a unix or linux command line (the '$' below represents the unix prompt), run the following command:  
+
+```
+$ curl --basic -u 'username:password' -H 'Accept: application/json' https://index.affectiva.com/
```
Comment (ahamino) [inline pages/eaasdocs/AffdexAPIOverview.md]:
```shell
curl --basic -u 'username:password' -H 'Accept: application/json' https://index.affectiva.com/
```

File: pages/eaasdocs/AffdexAPIOverview.md [hunk 5]
```diff
@@ -0,0 +1,152 @@
...
+"project": "project service URL"
+...
+},
+"v0": {
+...
+"project": "project service URL"
+...
+}
+}
+</code>
```
Comment (ahamino) [inline pages/eaasdocs/AffdexAPIOverview.md]: Replace that code block with the markdown of this:

GET https://index.affectiva.com/ HTTP/1.1
Accept: application/json
Response:
{

   "development": {
      ...
      "project": "project service URL"
      ...
   },

   "v0": {
     ...
     "project": "project service URL"
     ...
   }

}

Surround with ```http and ``` .

File: pages/eaasdocs/AffdexAPIOverview.md [hunk 6]
```diff
@@ -0,0 +1,152 @@
...
+
+This document describes version 0 of the API, whose key in the index is "v0".  
+
+## Authentication  
+
+The API uses HTTP Basic Access Authentication to authenticate clients.  HTTP Basic Authentication relies on a TLS transport to ensure that the client credentials cannot be seen by a third party, so all API communications must be transmitted over a TLS-protected channel.  Please ensure that your Index Service API begins with “https” before using it.  
+Our implementation allows the client to use one of two different techniques to authenticate with the server.  Credentials can be inserted into the URL, for example: https://userid:password@index.affectiva.com/.  Affdex usernames typically include an @ character which needs to be url-encoded as %40.  The password also needs to be URL-encoded.  
+Credentials can also be encoded and sent in an Authorization header per section 10.2 of RFC 1945.  This technique is recommended and is typically well-supported by HTTP client programming toolkits.
```
Comment (ahamino) [inline pages/eaasdocs/AffdexAPIOverview.md]: surround https ... with code blocks and put it on it's own line

```
https://userid:password@index.affectiva.com/
```

File: pages/eaasdocs/AffdexAPIOverview.md (hunk 3 above)
Comment (BenjaminAnding) [inline pages/eaasdocs/AffdexAPIOverview.md]: the raw tag means to interpret the text within literally. 
I meant to backslash escape the pound sign, but the html took over! habit

**Comment by caboteria** [comment]
Thanks, Ben, I made a few tweaks and merged this manually.

**Pull request closed without merging**